WalletWasabi.Gui/Global.cs

```diff
@@ -144,7 +142,7 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 				await LegalChecker.InitializeAsync(HostedServices.Get<UpdateChecker>()).ConfigureAwait(false);
 				cancel.ThrowIfCancellationRequested();
 
-				SystemAwakeChecker? systemAwakeChecker = await SystemAwakeChecker.CreateAsync(CoinJoinManager).ConfigureAwait(false);
+				SystemAwakeChecker? systemAwakeChecker = await SystemAwakeChecker.CreateAsync(HostedServices.Get<CoinJoinManager>()).ConfigureAwait(false);
```

Comment:
We register `CJManager` at LOC177 with the `RegisterCoinJoinComponents()` function.
I'm afraid this will throw a `NullReferenceException`, but we can move this between `RegisterCoinJoinComponents()` and `await HostedServices.StartAllAsync(cancel).ConfigureAwait(false)`.

Reply:
Fixed!

Reply:
Don't forget to push it up 😄